static/telemetry/graph.js

```diff
@@ -0,0 +1,85 @@
+/* global fetch */
+import MG from 'metrics-graphics';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default class TelemetryContainer extends React.Component {
+  async componentDidMount() {
+    const { id } = this.props;
+    this.fetchPlotGraph(id);
+  }
+
+  async fetchPlotGraph(id) {
+    const { graphData, telemetryUrl } = await (
+      await fetch(`/api/perf/telemetry/${id}`)).json();
+    this.graphTitleLink.setAttribute('href', telemetryUrl);
+    this.graphSubtitleEl.textContent = graphData.description;
+    this.graphEvolutionsTimeline(graphData, this.graphEl);
+  }
+
+  fixDates(datas) {
+    // When we fetch the backend for telemetry data the dates have been
+    // converted to strings. In here we fix them up
+    // XXX: This kind of operation has already happened in the backend, however,
+    // when we query the api we call json() and that turns object into
+    // string representations.
+    return datas.map(evo => evo.map(({ date, value }) => {
+      const YYYY = date.substring(0, 4);
+      const MM = date.substring(5, 7);
+      const DD = date.substring(8, 10);
+      const newDate = new Date(`${YYYY}-${MM}-${DD}`);
+      return {
+        date: newDate,
+        value,
+      };
+    }));
+  }
+
+  graphEvolutionsTimeline({ datas, params, yLabel, legendLabels }, graphEl) {
+    const newDatas = this.fixDates(datas);
```

Comment:
You can probably use `MG.convert.date`, like [here](https://github.com/mozilla/platform-health/blob/4fb7e6e03c90ac82780025d9e7eab99ad0f69b8d/static/crashes/xp.js#L10).